ledfx/effects/twod.py

```diff
@@ -215,7 +213,8 @@ def draw(self):
     def render(self):
         if self.init:
             self.do_once()
-
+        # Update the time every frame
+        self.current_time = timeit.default_timer()
```

Comment:
Moved from log_sec to here - this means current_time is updated every frame render I believe

Reply:
If you want it here, which I can understand, I would put it above if self.init, so we know its the first thing that happens in a render call